src/en/news/blog/2022/autoscaler_tuning/index.md

```diff
@@ -0,0 +1,143 @@
+---
+title: "Everything you need to know about the PG Autoscaler before upgrading to Quincy"
+date: "2022-06-01"
+author: "Laura Flores and Kamoltat Sirivadhna"
+tags:
+  - "autoscaler"
+  - "mgr"
+  - "quincy"
+  - "upgrade"
+  - "tuning"
+  - "troubleshooting"
+  - "management"
+  - "ceph"
+  - "performance"
+---
+
+## Introduction
+
+The pg\_autoscaler module, first introduced in the Nautilus (14.2.x) release, is an excellent way to automatically manage your Ceph cluster. Based on expected pool usage, the pg\_autoscaler can make recommendations and adjust the number of placement groups in a cluster based on pool usage and tunings set by the user.
+
+The autoscaler comes with a selection of tuning options, including the `--bulk` flag, `pg_num`, `target_size_ratio`, `noautoscale`, and `bias`. In preparation for the latest Quincy release, we put these tunings through release criteria testing and derived tuning recommendations and troubleshooting tips that users can reference as they upgrade to Quincy. In this blog post, we summarize initial challenges posed by the autoscaler, outline best tuning practices that came as a result of release criteria testing, and end with some autoscaler troubleshooting tips.
```

Comment:
maybe we should also mention pg_num_min and pg_num_max.

Reply:
yep that make sense